connect/transforms/src/main/java/org/apache/kafka/connect/transforms/TimestampConverter.java

```diff
@@ -50,6 +52,7 @@
 import static org.apache.kafka.connect.transforms.util.Requirements.requireStruct;
 
 public abstract class TimestampConverter<R extends ConnectRecord<R>> implements Transformation<R> {
+    private static final Logger log = LoggerFactory.getLogger(TimestampConverter.class);
```

Comment:
I'm not sure any of these log messages add value since TransformationChain has log messages with the records, which should be sufficient to know whether the schemas are null.